src/pages/journal/journal.scss

```diff
@@ -2,4 +2,14 @@ page-journal {
   h3 {
     text-align: center;
   }
+  .last-sync {
+    color: #dbd8d8;
```

Comment:
Can we use the GDS colours array for setting the this variable please?

Reply:
I'm not sure this is a GDS colour, as I don't see this value in `$colors-gds`. That said, I don't know where the values for `$colors-gds` came from, it could be that some standard colours are missing.

I considered adding to `$colors`, but didn't bother in the end as I didn't foresee the colour being reused - are we mandating that _all_ colours are added as SASS variables? Personally, I'd prefer the approach of using colour codes directly and doing a quick search through the codebase for existing usages whenever you introduce a colour (at which point you can extract a variable). Thoughts?

Reply:
I think all colours should be reused, and we should have a colour palette. There is no need for colours being used only in one place.

But let's leave this as it is right now, and have a discussion with Nad and Oli. They will be able to talk us through the colour palette. And by doing that I'm pretty sure we can remove quite a few of the already registered colours in `$colors-gds`.

Reply:
Also I'm not sure if new colours should be introduced just like that outside of that colour palette that we are going to establish with the designers.